deploy/helm/sumologic/templates/setup/setup-configmap.yaml

```diff
@@ -0,0 +1,143 @@
+{{- if .Values.sumologic.setupEnabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name:  {{ template "sumologic.fullname" . }}-setup
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "2"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+  labels:
+    app: {{ template "sumologic.labels.app" . }}
+    {{ template "sumologic.labels.common" . }}
+data:
+  setup.sh: |-
+    #!/bin/sh
+
+    mkdir /test
+    cp /etc/terraform/* /test/
+    cd /test
```

Comment:
L17-19 is because of [this issue](https://wiki.onap.org/pages/viewpage.action?pageId=29787473) with mounted volume